dredis/server.py

```diff
@@ -194,6 +198,9 @@ def main():
         logger.info("Finished loading (%.2f seconds)." % (time.time() - start_time))
 
     RedisServer(args.host, args.port)
+    gc_thread = gc.KeyGarbageCollector(args.gc_interval, args.gc_batch_size)
+    gc_thread.setDaemon(True)
```

Comment:
Small thing — was curious what this did and when researching, learned that this syntax is [legacy and will probably be deprecated](https://docs.python.org/3.7/library/threading.html#threading.Thread.daemon).

Their docs suggest:
```python
gc_thread.daemon = True
gc_thread.start()
```

Reply:
Python 2.7 doesn't support the parameter :-(

Reply:
Ah right, it's 2.7 — [their 2.7 docs](https://docs.python.org/2.7/library/threading.html#threading.Thread.daemon) say it does, but does it not work?

Reply:
@jimjshields: the property is available, but `__init__` doesn't take it:

```
>>> import threading
>>> t = threading.Thread(daemon=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: __init__() got an unexpected keyword argument 'daemon'
```

Reply:
Oops, I thought you were talking about passing `daemon` to `__init__` instead of replacing the `setDaemon` call. Nevermind, I can change the code to use the property setter.